### DISCUSSION TITLE
Simulation MVP Requirements

### DISCUSSION THREAD

**Discussion by Quantum1000** [Comet-Robotics/URC#20] [Simulation]
Minimum Viable Product:
- Able to render 2 views out (maybe use VR rendering mode?)
- Physics to simulate a simple rover
- Simple terrain
- API to interface with ROS and other code

**Comment by pablo-collantes** [comment]
Looking at other teams like West Virginia University they made their [simulation](https://github.com/team-mountaineers-urc-2024/simulation2024) through Gazebo and they used this [template](https://github.com/gazebosim/ros_gz_project_template) to add ROS2.
